templates/cis.auditd.conf.erb

```diff
@@ -0,0 +1,35 @@
+#
+# This file is managed using Chef. Any changes will be overwritten.
+#
+# This file controls the configuration of the audit daemon
+#
+
+local_events = yes
+write_logs = yes
+log_file = /var/log/audit/audit.log
+log_group = root
+log_format = RAW
+flush = INCREMENTAL_ASYNC
+freq = 50
+max_log_file = 8
+num_logs = 5
+priority_boost = 4
+disp_qos = lossy
+dispatcher = /sbin/audispd
+name_format = NONE
+max_log_file_action = keep_logs
+space_left = 75
+space_left_action = email
+action_mail_acct = root
+admin_space_left = 50
+admin_space_left_action = halt
+disk_full_action = SUSPEND
+disk_error_action = SUSPEND
```

Comment:
Having the entire file Chef-managed will prevent users from being able to adjust any of these settings. I'm not sure anyone is likely to want/need to adjust any of these though.
I think this change is good - if anyone wants any of these adjustable, we can add attributes later.

It would also be cool (but again, can do it later) to add comments to any lines of config that map to a CIS recommendation